From 7a66d79504a8a3b7a79a717312a7679f9ddf0686 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hern=C3=A1n=20Casta=C3=B1=C3=B3n=20=C3=81lvarez?=
 <herna@paranoidandroid.co>
Date: Tue, 1 Oct 2019 14:36:12 +0000
Subject: [PATCH] SystemUI: Update Bluetooth battery level assets
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Outlined and polished. Designed by Andrew Fluck.

Change-Id: I94fd78d5788306f64623e850a00302bb42b7897d
Signed-off-by: Hernán Castañón Álvarez <herna@paranoidandroid.co>
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 ...sys_data_bluetooth_connected_battery_0.xml | 57 ++++---------------
 ...sys_data_bluetooth_connected_battery_1.xml | 57 ++++---------------
 ...sys_data_bluetooth_connected_battery_2.xml | 57 ++++---------------
 ...sys_data_bluetooth_connected_battery_3.xml | 57 ++++---------------
 ...sys_data_bluetooth_connected_battery_4.xml | 57 ++++---------------
 ...sys_data_bluetooth_connected_battery_5.xml | 57 ++++---------------
 ...sys_data_bluetooth_connected_battery_6.xml | 57 ++++---------------
 ...sys_data_bluetooth_connected_battery_7.xml | 57 ++++---------------
 ...sys_data_bluetooth_connected_battery_8.xml | 57 ++++---------------
 ...sys_data_bluetooth_connected_battery_9.xml | 53 ++++-------------
 10 files changed, 119 insertions(+), 447 deletions(-)

diff --git a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_0.xml b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_0.xml
index 3559e09db09b..a5a5c7dd55d6 100644
--- a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_0.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_0.xml
@@ -1,47 +1,14 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-**
-** Copyright 2017, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License");
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
--->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:width="20dp"
-        android:height="17dp"
-        android:viewportWidth="21.0"
-        android:viewportHeight="18.0">
-    <group
-        android:scaleX="0.75"
-        android:scaleY="0.75">
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M17.71,7.71L12,2h-1v7.59L6.41,5 5,6.41 10.59,12 5,17.59 6.41,19 11,14.41L11,22h1l5.71,-5.71 -4.3,-4.29 4.3,-4.29zM13,5.83l1.88,1.88L13,9.59L13,5.83zM14.88,16.29L13,18.17v-3.76l1.88,1.88z"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M5,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M19,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-    </group>
-    <group
-        android:translateY="0.5"
-        android:translateX="0.5" >
-        <path
-            android:pathData="M15.77,1.064V15.94h3V1.064Z"
-            android:fillColor="#4DFFFFFF"/>
-        <path
-            android:pathData="M15.77,15.3V15.94h3V15.3Z"
-            android:fillColor="#FFFFFF"/>
-    </group>
+    android:width="20dp"
+    android:height="17dp"
+    android:viewportWidth="20"
+    android:viewportHeight="17">
+  <path
+      android:pathData="M14.4583,1.4167C14.0671,1.4167 13.75,1.7338 13.75,2.125V14.875C13.75,15.2662 14.0671,15.5833 14.4583,15.5833H17.2917C17.6829,15.5833 18,15.2662 18,14.875V2.125C18,1.7338 17.6829,1.4167 17.2917,1.4167H14.4583ZM15.1667,2.8333H16.5833V13.3167H15.1667V2.8333Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M11.0046,5.4613L6.9601,1.4167H6.2517V6.7929L3.0005,3.5417L2.0017,4.5404L5.9613,8.5L2.0017,12.4596L3.0005,13.4583L6.2517,10.2071V15.5833H6.9601L11.0046,11.5388L7.9588,8.5L11.0046,5.4613ZM3.4184,7.7917C3.0272,7.4005 2.3929,7.4005 2.0017,7.7917C1.6105,8.1829 1.6105,8.8171 2.0017,9.2083C2.3929,9.5995 3.0272,9.5995 3.4184,9.2083C3.8096,8.8171 3.8096,8.1829 3.4184,7.7917ZM9.0001,5.4613L7.6684,4.1296V6.7929L9.0001,5.4613ZM9.0001,11.5388L7.6684,12.8704V10.2071L9.0001,11.5388ZM10.5017,7.7917C10.8929,7.4005 11.5272,7.4005 11.9184,7.7917C12.3096,8.1829 12.3096,8.8171 11.9184,9.2083C11.5272,9.5995 10.8929,9.5995 10.5017,9.2083C10.1105,8.8171 10.1105,8.1829 10.5017,7.7917Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
 </vector>
diff --git a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_1.xml b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_1.xml
index 50c9438c707d..7af2d7309add 100644
--- a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_1.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_1.xml
@@ -1,47 +1,14 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-**
-** Copyright 2017, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License");
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
--->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:width="20dp"
-        android:height="17dp"
-        android:viewportWidth="21.0"
-        android:viewportHeight="18.0">
-    <group
-        android:scaleX="0.75"
-        android:scaleY="0.75">
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M17.71,7.71L12,2h-1v7.59L6.41,5 5,6.41 10.59,12 5,17.59 6.41,19 11,14.41L11,22h1l5.71,-5.71 -4.3,-4.29 4.3,-4.29zM13,5.83l1.88,1.88L13,9.59L13,5.83zM14.88,16.29L13,18.17v-3.76l1.88,1.88z"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M5,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M19,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-    </group>
-    <group
-        android:translateY="0.5"
-        android:translateX="0.5" >
-        <path
-            android:pathData="M15.77,1.064V15.94h3V1.064Z"
-            android:fillColor="#4DFFFFFF"/>
-        <path
-            android:pathData="M15.77,13.6V15.94h3V13.6Z"
-            android:fillColor="#FFFFFF"/>
-    </group>
+    android:width="20dp"
+    android:height="17dp"
+    android:viewportWidth="20"
+    android:viewportHeight="17">
+  <path
+      android:pathData="M11.0046,5.4613L6.9601,1.4167H6.2517V6.7929L3.0005,3.5417L2.0017,4.5404L5.9613,8.5L2.0017,12.4596L3.0005,13.4583L6.2517,10.2071V15.5833H6.9601L11.0046,11.5388L7.9588,8.5L11.0046,5.4613ZM3.4184,7.7917C3.0272,7.4005 2.3929,7.4005 2.0017,7.7917C1.6105,8.1829 1.6105,8.8171 2.0017,9.2083C2.3929,9.5995 3.0272,9.5995 3.4184,9.2083C3.8096,8.8171 3.8096,8.1829 3.4184,7.7917ZM9.0001,5.4613L7.6684,4.1296V6.7929L9.0001,5.4613ZM9.0001,11.5388L7.6684,12.8704V10.2071L9.0001,11.5388ZM10.5017,7.7917C10.8929,7.4005 11.5272,7.4005 11.9184,7.7917C12.3096,8.1829 12.3096,8.8171 11.9184,9.2083C11.5272,9.5995 10.8929,9.5995 10.5017,9.2083C10.1105,8.8171 10.1105,8.1829 10.5017,7.7917Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M14.4583,1.4167C14.0671,1.4167 13.75,1.7338 13.75,2.125V14.875C13.75,15.2662 14.0671,15.5833 14.4583,15.5833H17.2917C17.6829,15.5833 18,15.2662 18,14.875V2.125C18,1.7338 17.6829,1.4167 17.2917,1.4167H14.4583ZM15.1667,2.8333H16.5833V12.1833H15.1667V2.8333Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
 </vector>
diff --git a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_2.xml b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_2.xml
index 0c82aa5c65ed..f4c6655f5495 100644
--- a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_2.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_2.xml
@@ -1,47 +1,14 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-**
-** Copyright 2017, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License");
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
--->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:width="20dp"
-        android:height="17dp"
-        android:viewportWidth="21.0"
-        android:viewportHeight="18.0">
-    <group
-        android:scaleX="0.75"
-        android:scaleY="0.75">
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M17.71,7.71L12,2h-1v7.59L6.41,5 5,6.41 10.59,12 5,17.59 6.41,19 11,14.41L11,22h1l5.71,-5.71 -4.3,-4.29 4.3,-4.29zM13,5.83l1.88,1.88L13,9.59L13,5.83zM14.88,16.29L13,18.17v-3.76l1.88,1.88z"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M5,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M19,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-    </group>
-    <group
-        android:translateY="0.5"
-        android:translateX="0.5" >
-        <path
-            android:pathData="M15.77,1.064V15.94h3V1.064Z"
-            android:fillColor="#4DFFFFFF"/>
-        <path
-            android:pathData="M15.77,11.9V15.94h3V11.9Z"
-            android:fillColor="#FFFFFF"/>
-    </group>
+    android:width="20dp"
+    android:height="17dp"
+    android:viewportWidth="20"
+    android:viewportHeight="17">
+  <path
+      android:pathData="M14.4583,1.4167C14.0671,1.4167 13.75,1.7338 13.75,2.125V14.875C13.75,15.2662 14.0671,15.5833 14.4583,15.5833H17.2917C17.6829,15.5833 18,15.2662 18,14.875V2.125C18,1.7338 17.6829,1.4167 17.2917,1.4167H14.4583ZM15.1667,2.8333H16.5833V10.2H15.1667V2.8333Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M11.0047,5.4613L6.9601,1.4167H6.2517V6.7929L3.0005,3.5417L2.0017,4.5404L5.9613,8.5L2.0017,12.4596L3.0005,13.4583L6.2517,10.2071V15.5833H6.9601L11.0047,11.5388L7.9588,8.5L11.0047,5.4613ZM3.4184,7.7917C3.0272,7.4005 2.3929,7.4005 2.0017,7.7917C1.6105,8.1829 1.6105,8.8171 2.0017,9.2083C2.3929,9.5995 3.0272,9.5995 3.4184,9.2083C3.8096,8.8171 3.8096,8.1829 3.4184,7.7917ZM9.0001,5.4613L7.6684,4.1296V6.7929L9.0001,5.4613ZM9.0001,11.5388L7.6684,12.8704V10.2071L9.0001,11.5388ZM10.5017,7.7917C10.8929,7.4005 11.5272,7.4005 11.9184,7.7917C12.3096,8.1829 12.3096,8.8171 11.9184,9.2083C11.5272,9.5995 10.8929,9.5995 10.5017,9.2083C10.1105,8.8171 10.1105,8.1829 10.5017,7.7917Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
 </vector>
diff --git a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_3.xml b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_3.xml
index 0f394873af2d..af20c4ff350c 100644
--- a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_3.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_3.xml
@@ -1,47 +1,14 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-**
-** Copyright 2017, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License");
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
--->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:width="20dp"
-        android:height="17dp"
-        android:viewportWidth="21.0"
-        android:viewportHeight="18.0">
-    <group
-        android:scaleX="0.75"
-        android:scaleY="0.75">
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M17.71,7.71L12,2h-1v7.59L6.41,5 5,6.41 10.59,12 5,17.59 6.41,19 11,14.41L11,22h1l5.71,-5.71 -4.3,-4.29 4.3,-4.29zM13,5.83l1.88,1.88L13,9.59L13,5.83zM14.88,16.29L13,18.17v-3.76l1.88,1.88z"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M5,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M19,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-    </group>
-    <group
-        android:translateY="0.5"
-        android:translateX="0.5" >
-        <path
-            android:pathData="M15.77,1.064V15.94h3V1.064Z"
-            android:fillColor="#4DFFFFFF"/>
-        <path
-            android:pathData="M15.77,10.2V15.94h3V10.2Z"
-            android:fillColor="#FFFFFF"/>
-    </group>
+    android:width="20dp"
+    android:height="17dp"
+    android:viewportWidth="20"
+    android:viewportHeight="17">
+  <path
+      android:pathData="M14.4583,1.4167C14.0671,1.4167 13.75,1.7338 13.75,2.125V14.875C13.75,15.2662 14.0671,15.5833 14.4583,15.5833H17.2917C17.6829,15.5833 18,15.2662 18,14.875V2.125C18,1.7338 17.6829,1.4167 17.2917,1.4167H14.4583ZM16.5833,2.8333H15.1667V9.35H16.5833V2.8333Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M11.0047,5.4613L6.9601,1.4167H6.2517V6.7929L3.0005,3.5417L2.0017,4.5404L5.9613,8.5L2.0017,12.4596L3.0005,13.4583L6.2517,10.2071V15.5833H6.9601L11.0047,11.5388L7.9588,8.5L11.0047,5.4613ZM3.4184,7.7917C3.0272,7.4005 2.3929,7.4005 2.0017,7.7917C1.6105,8.1829 1.6105,8.8171 2.0017,9.2083C2.3929,9.5995 3.0272,9.5995 3.4184,9.2083C3.8096,8.8171 3.8096,8.1829 3.4184,7.7917ZM9.0001,5.4613L7.6684,4.1296V6.7929L9.0001,5.4613ZM9.0001,11.5388L7.6684,12.8704V10.2071L9.0001,11.5388ZM10.5017,7.7917C10.8929,7.4005 11.5272,7.4005 11.9184,7.7917C12.3096,8.1829 12.3096,8.8171 11.9184,9.2083C11.5272,9.5995 10.8929,9.5995 10.5017,9.2083C10.1105,8.8171 10.1105,8.1829 10.5017,7.7917Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
 </vector>
diff --git a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_4.xml b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_4.xml
index 6a613d9f625e..04c5b18d55d3 100644
--- a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_4.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_4.xml
@@ -1,47 +1,14 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-**
-** Copyright 2017, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License");
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
--->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:width="20dp"
-        android:height="17dp"
-        android:viewportWidth="21.0"
-        android:viewportHeight="18.0">
-    <group
-        android:scaleX="0.75"
-        android:scaleY="0.75">
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M17.71,7.71L12,2h-1v7.59L6.41,5 5,6.41 10.59,12 5,17.59 6.41,19 11,14.41L11,22h1l5.71,-5.71 -4.3,-4.29 4.3,-4.29zM13,5.83l1.88,1.88L13,9.59L13,5.83zM14.88,16.29L13,18.17v-3.76l1.88,1.88z"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M5,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M19,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-    </group>
-    <group
-        android:translateY="0.5"
-        android:translateX="0.5" >
-        <path
-            android:pathData="M15.77,1.064V15.94h3V1.064Z"
-            android:fillColor="#4DFFFFFF"/>
-        <path
-            android:pathData="M15.77,8.5V15.94h3V8.5Z"
-            android:fillColor="#FFFFFF"/>
-    </group>
+    android:width="20dp"
+    android:height="17dp"
+    android:viewportWidth="20"
+    android:viewportHeight="17">
+  <path
+      android:pathData="M14.4583,1.4167C14.0671,1.4167 13.75,1.7338 13.75,2.125V14.875C13.75,15.2662 14.0671,15.5833 14.4583,15.5833H17.2917C17.6829,15.5833 18,15.2662 18,14.875V2.125C18,1.7338 17.6829,1.4167 17.2917,1.4167H14.4583ZM15.1667,2.8333H16.5833V7.0833H15.1667V2.8333Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M11.0046,5.4613L6.9601,1.4167H6.2517V6.7929L3.0005,3.5417L2.0017,4.5404L5.9613,8.5L2.0017,12.4596L3.0005,13.4583L6.2517,10.2071V15.5833H6.9601L11.0046,11.5388L7.9588,8.5L11.0046,5.4613ZM3.4184,7.7917C3.0272,7.4005 2.3929,7.4005 2.0017,7.7917C1.6105,8.1829 1.6105,8.8171 2.0017,9.2083C2.3929,9.5995 3.0272,9.5995 3.4184,9.2083C3.8096,8.8171 3.8096,8.1829 3.4184,7.7917ZM9.0001,5.4613L7.6684,4.1296V6.7929L9.0001,5.4613ZM9.0001,11.5388L7.6684,12.8704V10.2071L9.0001,11.5388ZM10.5017,7.7917C10.8929,7.4005 11.5272,7.4005 11.9184,7.7917C12.3096,8.1829 12.3096,8.8171 11.9184,9.2083C11.5272,9.5995 10.8929,9.5995 10.5017,9.2083C10.1105,8.8171 10.1105,8.1829 10.5017,7.7917Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
 </vector>
diff --git a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_5.xml b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_5.xml
index df0f93d9e026..964d2e173fc0 100644
--- a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_5.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_5.xml
@@ -1,47 +1,14 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-**
-** Copyright 2017, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License");
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
--->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:width="20dp"
-        android:height="17dp"
-        android:viewportWidth="21.0"
-        android:viewportHeight="18.0">
-    <group
-        android:scaleX="0.75"
-        android:scaleY="0.75">
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M17.71,7.71L12,2h-1v7.59L6.41,5 5,6.41 10.59,12 5,17.59 6.41,19 11,14.41L11,22h1l5.71,-5.71 -4.3,-4.29 4.3,-4.29zM13,5.83l1.88,1.88L13,9.59L13,5.83zM14.88,16.29L13,18.17v-3.76l1.88,1.88z"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M5,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M19,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-    </group>
-    <group
-        android:translateY="0.5"
-        android:translateX="0.5" >
-        <path
-            android:pathData="M15.77,1.064V15.94h3V1.064Z"
-            android:fillColor="#4DFFFFFF"/>
-        <path
-            android:pathData="M15.77,6.8V15.94h3V6.8Z"
-            android:fillColor="#FFFFFF"/>
-    </group>
+    android:width="20dp"
+    android:height="17dp"
+    android:viewportWidth="20"
+    android:viewportHeight="17">
+  <path
+      android:pathData="M14.4583,1.4167C14.0671,1.4167 13.75,1.7338 13.75,2.125V14.875C13.75,15.2662 14.0671,15.5833 14.4583,15.5833H17.2917C17.6829,15.5833 18,15.2662 18,14.875V2.125C18,1.7338 17.6829,1.4167 17.2917,1.4167H14.4583ZM15.1667,2.8333H16.5833V6.2333H15.1667V2.8333Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M11.0046,5.4613L6.9601,1.4167H6.2517V6.7929L3.0005,3.5417L2.0017,4.5404L5.9613,8.5L2.0017,12.4596L3.0005,13.4583L6.2517,10.2071V15.5833H6.9601L11.0046,11.5388L7.9588,8.5L11.0046,5.4613ZM3.4184,7.7917C3.0272,7.4005 2.3929,7.4005 2.0017,7.7917C1.6105,8.1829 1.6105,8.8171 2.0017,9.2083C2.3929,9.5995 3.0272,9.5995 3.4184,9.2083C3.8096,8.8171 3.8096,8.1829 3.4184,7.7917ZM9.0001,5.4613L7.6684,4.1296V6.7929L9.0001,5.4613ZM9.0001,11.5388L7.6684,12.8704V10.2071L9.0001,11.5388ZM10.5017,7.7917C10.8929,7.4005 11.5272,7.4005 11.9184,7.7917C12.3096,8.1829 12.3096,8.8171 11.9184,9.2083C11.5272,9.5995 10.8929,9.5995 10.5017,9.2083C10.1105,8.8171 10.1105,8.1829 10.5017,7.7917Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
 </vector>
diff --git a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_6.xml b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_6.xml
index 4924deada67d..7599f9a89faf 100644
--- a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_6.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_6.xml
@@ -1,47 +1,14 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-**
-** Copyright 2017, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License");
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
--->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:width="20dp"
-        android:height="17dp"
-        android:viewportWidth="21.0"
-        android:viewportHeight="18.0">
-    <group
-        android:scaleX="0.75"
-        android:scaleY="0.75">
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M17.71,7.71L12,2h-1v7.59L6.41,5 5,6.41 10.59,12 5,17.59 6.41,19 11,14.41L11,22h1l5.71,-5.71 -4.3,-4.29 4.3,-4.29zM13,5.83l1.88,1.88L13,9.59L13,5.83zM14.88,16.29L13,18.17v-3.76l1.88,1.88z"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M5,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M19,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-    </group>
-    <group
-        android:translateY="0.5"
-        android:translateX="0.5" >
-        <path
-            android:pathData="M15.77,1.064V15.94h3V1.064Z"
-            android:fillColor="#4DFFFFFF"/>
-        <path
-            android:pathData="M15.77,5.1V15.94h3V5.1Z"
-            android:fillColor="#FFFFFF"/>
-    </group>
+    android:width="20dp"
+    android:height="17dp"
+    android:viewportWidth="20"
+    android:viewportHeight="17">
+  <path
+      android:pathData="M14.4583,1.4167C14.0671,1.4167 13.75,1.7338 13.75,2.125V14.875C13.75,15.2662 14.0671,15.5833 14.4583,15.5833H17.2917C17.6829,15.5833 18,15.2662 18,14.875V2.125C18,1.7338 17.6829,1.4167 17.2917,1.4167H14.4583ZM15.1667,2.8333H16.5833V5.3833H15.1667V2.8333Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M11.0046,5.4613L6.9601,1.4167H6.2517V6.7929L3.0005,3.5417L2.0017,4.5404L5.9613,8.5L2.0017,12.4596L3.0005,13.4583L6.2517,10.2071V15.5833H6.9601L11.0046,11.5388L7.9588,8.5L11.0046,5.4613ZM3.4184,7.7917C3.0272,7.4005 2.3929,7.4005 2.0017,7.7917C1.6105,8.1829 1.6105,8.8171 2.0017,9.2083C2.3929,9.5995 3.0272,9.5995 3.4184,9.2083C3.8096,8.8171 3.8096,8.1829 3.4184,7.7917ZM9.0001,5.4613L7.6684,4.1296V6.7929L9.0001,5.4613ZM9.0001,11.5388L7.6684,12.8704V10.2071L9.0001,11.5388ZM10.5017,7.7917C10.8929,7.4005 11.5272,7.4005 11.9184,7.7917C12.3096,8.1829 12.3096,8.8171 11.9184,9.2083C11.5272,9.5995 10.8929,9.5995 10.5017,9.2083C10.1105,8.8171 10.1105,8.1829 10.5017,7.7917Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
 </vector>
diff --git a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_7.xml b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_7.xml
index 4c4ae035aa23..9017d36ad4f3 100644
--- a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_7.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_7.xml
@@ -1,47 +1,14 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-**
-** Copyright 2017, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License");
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
--->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:width="20dp"
-        android:height="17dp"
-        android:viewportWidth="21.0"
-        android:viewportHeight="18.0">
-    <group
-        android:scaleX="0.75"
-        android:scaleY="0.75">
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M17.71,7.71L12,2h-1v7.59L6.41,5 5,6.41 10.59,12 5,17.59 6.41,19 11,14.41L11,22h1l5.71,-5.71 -4.3,-4.29 4.3,-4.29zM13,5.83l1.88,1.88L13,9.59L13,5.83zM14.88,16.29L13,18.17v-3.76l1.88,1.88z"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M5,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M19,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-    </group>
-    <group
-        android:translateY="0.5"
-        android:translateX="0.5" >
-        <path
-            android:pathData="M15.77,1.064V15.94h3V1.064Z"
-            android:fillColor="#4DFFFFFF"/>
-        <path
-            android:pathData="M15.77,3.4V15.94h3V3.4Z"
-            android:fillColor="#FFFFFF"/>
-    </group>
+    android:width="20dp"
+    android:height="17dp"
+    android:viewportWidth="20"
+    android:viewportHeight="17">
+  <path
+      android:pathData="M14.4583,1.4167C14.0671,1.4167 13.75,1.7338 13.75,2.125V14.875C13.75,15.2662 14.0671,15.5833 14.4583,15.5833H17.2917C17.6829,15.5833 18,15.2662 18,14.875V2.125C18,1.7338 17.6829,1.4167 17.2917,1.4167H14.4583ZM15.1667,2.8333H16.5833V4.5333H15.1667V2.8333Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M11.0046,5.4613L6.9601,1.4167H6.2517V6.7929L3.0005,3.5417L2.0017,4.5404L5.9613,8.5L2.0017,12.4596L3.0005,13.4583L6.2517,10.2071V15.5833H6.9601L11.0046,11.5388L7.9588,8.5L11.0046,5.4613ZM3.4184,7.7917C3.0272,7.4005 2.3929,7.4005 2.0017,7.7917C1.6105,8.1829 1.6105,8.8171 2.0017,9.2083C2.3929,9.5995 3.0272,9.5995 3.4184,9.2083C3.8096,8.8171 3.8096,8.1829 3.4184,7.7917ZM9.0001,5.4613L7.6684,4.1296V6.7929L9.0001,5.4613ZM9.0001,11.5388L7.6684,12.8704V10.2071L9.0001,11.5388ZM10.5017,7.7917C10.8929,7.4005 11.5272,7.4005 11.9184,7.7917C12.3096,8.1829 12.3096,8.8171 11.9184,9.2083C11.5272,9.5995 10.8929,9.5995 10.5017,9.2083C10.1105,8.8171 10.1105,8.1829 10.5017,7.7917Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
 </vector>
diff --git a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_8.xml b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_8.xml
index 6ba978aee111..06de4b3acd88 100644
--- a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_8.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_8.xml
@@ -1,47 +1,14 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-**
-** Copyright 2017, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License");
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
--->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:width="20dp"
-        android:height="17dp"
-        android:viewportWidth="21.0"
-        android:viewportHeight="18.0">
-    <group
-        android:scaleX="0.75"
-        android:scaleY="0.75">
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M17.71,7.71L12,2h-1v7.59L6.41,5 5,6.41 10.59,12 5,17.59 6.41,19 11,14.41L11,22h1l5.71,-5.71 -4.3,-4.29 4.3,-4.29zM13,5.83l1.88,1.88L13,9.59L13,5.83zM14.88,16.29L13,18.17v-3.76l1.88,1.88z"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M5,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M19,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-    </group>
-    <group
-        android:translateY="0.5"
-        android:translateX="0.5" >
-        <path
-            android:pathData="M15.77,1.064V15.94h3V1.064Z"
-            android:fillColor="#4DFFFFFF"/>
-        <path
-            android:pathData="M15.77,1.7V15.94h3V1.7Z"
-            android:fillColor="#FFFFFF"/>
-    </group>
+    android:width="20dp"
+    android:height="17dp"
+    android:viewportWidth="20"
+    android:viewportHeight="17">
+  <path
+      android:pathData="M14.4583,1.4167C14.0671,1.4167 13.75,1.7338 13.75,2.125V14.875C13.75,15.2662 14.0671,15.5833 14.4583,15.5833H17.2917C17.6829,15.5833 18,15.2662 18,14.875V2.125C18,1.7338 17.6829,1.4167 17.2917,1.4167H14.4583ZM15.1667,2.8333H16.5833V3.6833H15.1667V2.8333Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M11.0047,5.4613L6.9601,1.4167H6.2517V6.7929L3.0005,3.5417L2.0017,4.5404L5.9613,8.5L2.0017,12.4596L3.0005,13.4583L6.2517,10.2071V15.5833H6.9601L11.0047,11.5388L7.9588,8.5L11.0047,5.4613ZM3.4184,7.7917C3.0272,7.4005 2.393,7.4005 2.0017,7.7917C1.6105,8.1829 1.6105,8.8171 2.0017,9.2083C2.393,9.5995 3.0272,9.5995 3.4184,9.2083C3.8096,8.8171 3.8096,8.1829 3.4184,7.7917ZM9.0001,5.4613L7.6684,4.1296V6.7929L9.0001,5.4613ZM9.0001,11.5388L7.6684,12.8704V10.2071L9.0001,11.5388ZM10.5017,7.7917C10.8929,7.4005 11.5272,7.4005 11.9184,7.7917C12.3096,8.1829 12.3096,8.8171 11.9184,9.2083C11.5272,9.5995 10.8929,9.5995 10.5017,9.2083C10.1105,8.8171 10.1105,8.1829 10.5017,7.7917Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
 </vector>
diff --git a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_9.xml b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_9.xml
index 26ff151418a6..a0595229d118 100644
--- a/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_9.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_data_bluetooth_connected_battery_9.xml
@@ -1,44 +1,13 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-**
-** Copyright 2017, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License");
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
--->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:width="20dp"
-        android:height="17dp"
-        android:viewportWidth="21.0"
-        android:viewportHeight="18.0">
-    <group
-        android:scaleX="0.75"
-        android:scaleY="0.75">
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M17.71,7.71L12,2h-1v7.59L6.41,5 5,6.41 10.59,12 5,17.59 6.41,19 11,14.41L11,22h1l5.71,-5.71 -4.3,-4.29 4.3,-4.29zM13,5.83l1.88,1.88L13,9.59L13,5.83zM14.88,16.29L13,18.17v-3.76l1.88,1.88z"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M5,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-        <path
-            android:fillColor="@android:color/white"
-            android:pathData="M19,12m-1.5,0a1.5,1.5 0,1 1,3 0a1.5,1.5 0,1 1,-3 0"/>
-    </group>
-    <group
-        android:translateY="0.5"
-        android:translateX="0.5" >
-        <path
-            android:pathData="M15.77,1.064V15.94h3V1.064Z"
-            android:fillColor="#FFFFFF"/>
-    </group>
+    android:width="20dp"
+    android:height="17dp"
+    android:viewportWidth="20"
+    android:viewportHeight="17">
+  <path
+      android:pathData="M13.75,2.125C13.75,1.7338 14.0671,1.4167 14.4583,1.4167H17.2917C17.6829,1.4167 18,1.7338 18,2.125V14.875C18,15.2662 17.6829,15.5833 17.2917,15.5833H14.4583C14.0671,15.5833 13.75,15.2662 13.75,14.875V2.125Z"
+      android:fillColor="#000000"/>
+  <path
+      android:pathData="M11.0047,5.4613L6.9601,1.4167H6.2517V6.7929L3.0005,3.5417L2.0017,4.5404L5.9613,8.5L2.0017,12.4596L3.0005,13.4583L6.2517,10.2071V15.5833H6.9601L11.0047,11.5388L7.9588,8.5L11.0047,5.4613ZM3.4184,7.7917C3.0272,7.4005 2.393,7.4005 2.0017,7.7917C1.6105,8.1829 1.6105,8.8171 2.0017,9.2083C2.393,9.5995 3.0272,9.5995 3.4184,9.2083C3.8096,8.8171 3.8096,8.1829 3.4184,7.7917ZM9.0001,5.4613L7.6684,4.1296V6.7929L9.0001,5.4613ZM9.0001,11.5388L7.6684,12.8704V10.2071L9.0001,11.5388ZM10.5017,7.7917C10.8929,7.4005 11.5272,7.4005 11.9184,7.7917C12.3096,8.1829 12.3096,8.8171 11.9184,9.2083C11.5272,9.5995 10.8929,9.5995 10.5017,9.2083C10.1105,8.8171 10.1105,8.1829 10.5017,7.7917Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
 </vector>
-- 
2.49.0

